HashTable/hashtable.c

```diff
@@ -0,0 +1,145 @@
+#include "../List/list.h"
+#include <stdlib.h>
+#include "hashtable.h"
+#include <stdio.h>
+#include <string.h>
+#define TABLE_SIZE 1000
+
+typedef struct HT {
+    Node* elements[TABLE_SIZE];
+} HashTable;
+
+HashTable* initTable(void) {
+    HashTable* table = malloc(sizeof(HashTable));
+    if (table == NULL) {
+        perror("Memory allocate error");
+        return NULL;
+    }
+    for (int i = 0; i < TABLE_SIZE; ++i) {
+        (table->elements)[i] = NULL;
```

Comment:
```suggestion
        table->elements[i] = NULL;
```
Если бы выше был calloc, этого не надо было бы делать вручную